pkgs/development/python-modules/deepface/default.nix

```diff
@@ -0,0 +1,111 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fire,
+  flask,
+  flask-cors,
+  gdown,
+  gunicorn,
+  keras,
+  mtcnn,
+  numpy,
+  opencv4,
+  pandas,
+  pillow,
+  pythonOlder,
+  requests,
+  retinaface,
+  tensorflow,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "deepface";
+  version = "0.0.92";
+  format = "setuptools";
```

Comment:
Use `pyproject = true` and `build-system = [ setuptools ];` to avoid legacy build